test/WebJobs.Script.Tests/Workers/Rpc/GrpcWorkerChannelTests.cs

```diff
@@ -356,6 +356,31 @@ public void SendLoadRequests_PublishesOutboundEvents()
             Assert.True(functionLoadLogs.Count() == 2);
         }
 
+        [Fact]
+        public void SendLoadRequestCollection_PublishesOutboundEvents()
+        {
+            StartStream startStream = new StartStream()
+            {
+                WorkerId = _workerId
+            };
+            StreamingMessage startStreamMessage = new StreamingMessage()
+            {
+                StartStream = startStream
+            };
+            GrpcEvent rpcEvent = new GrpcEvent(_workerId, startStreamMessage);
```

Comment:
nit: can we add some new lines to clean this up for easier readability? not a big deal but it's nicer to read :)

``` 
// Arrange

// Act

// Assert